### ISSUE TITLE
Training on balloon dataset with CUDA 8 and CUDNN 6

### ISSUE THREAD

**Issue by Romiroz** [matterport/Mask_RCNN#2214]
Hello!
         When i try to train Mask_RCNN with balloon dataset i have the following error:

Caused by op 'fpn_p3/convolution', defined at:
  File "balloon.py", line 331, in <module>
    model_dir=args.logs)
  File "C:\Program Files\Python36\lib\site-packages\mrcnn\model.py", line 1832, in __init__
    self.keras_model = self.build(mode=mode, config=config)
  File "C:\Program Files\Python36\lib\site-packages\mrcnn\model.py", line 1919, in build
    P3 = KL.Conv2D(config.TOP_DOWN_PYRAMID_SIZE, (3, 3), padding="SAME", name="fpn_p3")(P3)
  File "C:\Program Files\Python36\lib\site-packages\keras\engine\topology.py", line 617, in __call__
    output = self.call(inputs, **kwargs)
  File "C:\Program Files\Python36\lib\site-packages\keras\layers\convolutional.py", line 168, in call
    dilation_rate=self.dilation_rate)
  File "C:\Program Files\Python36\lib\site-packages\keras\backend\tensorflow_backend.py", line 3332, in conv2d
    data_format=tf_data_format)
  File "C:\Program Files\Python36\lib\site-packages\tensorflow\python\ops\nn_ops.py", line 751, in convolution
    return op(input, filter)
  File "C:\Program Files\Python36\lib\site-packages\tensorflow\python\ops\nn_ops.py", line 835, in __call__
    return self.conv_op(inp, filter)
  File "C:\Program Files\Python36\lib\site-packages\tensorflow\python\ops\nn_ops.py", line 499, in __call__
    return self.call(inp, filter)
  File "C:\Program Files\Python36\lib\site-packages\tensorflow\python\ops\nn_ops.py", line 187, in __call__
    name=self.name)
  File "C:\Program Files\Python36\lib\site-packages\tensorflow\python\ops\gen_nn_ops.py", line 630, in conv2d
    data_format=data_format, name=name)
  File "C:\Program Files\Python36\lib\site-packages\tensorflow\python\framework\op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "C:\Program Files\Python36\lib\site-packages\tensorflow\python\framework\ops.py", line 2956, in create_op
    op_def=op_def)
  File "C:\Program Files\Python36\lib\site-packages\tensorflow\python\framework\ops.py", line 1470, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

InternalError (see above for traceback): cuDNN launch failure : input shape([2,256,128,128]) filter shape([3,3,256,256])
         [[Node: fpn_p3/convolution = Conv2D[T=DT_FLOAT, data_format="NHWC", padding="SAME", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:GPU:0"](fpn_p3add/add, fpn_p3/kernel/read)]]
         [[Node: mrcnn_bbox_loss/Reshape/_4571 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_6245_mrcnn_bbox_loss/Reshape", tensor_type=DT_INT32, _device="/job:localhost/replica:0/task:0/device:CPU:0"]()]]  

I have NVidia GRID K1 GPU. Do you have any solution? tf is 1.4 for this CUDA and CUDNN

**Comment by burhr2** [comment]
Hi! kindly see this issue https://github.com/matterport/Mask_RCNN/issues/2312 there is a pointer to the same implementation that supports TensorFlow 2=> (with GPU). Try that implementation.

If this solves your problem kindly close the issue so as others can navigate to other issues easier
